staging/operator-lifecycle-manager/test/e2e/catalog_e2e_test.go

```diff
@@ -47,9 +47,9 @@ const (
 
 var _ = Describe("Starting CatalogSource e2e tests", func() {
 	var (
-		ns  corev1.Namespace
-		c   operatorclient.ClientInterface
-		crc versioned.Interface
+		generatedNamespace corev1.Namespace
```

Comment:
Is this name change to ensure commonality between all the tests and the name of the variable they all use for the namespace? It's definitely more descriptive but it seems like the anticipated functionality is already present in this file without the rename.

Reply:
Hey, great callout! Your intuition is correct and this was discussed in the upstream PR. I should have introduced two commits:

- A commit that standardized the namespace variable name.
- A commit that introduced generated namespaces to e2e suites that didn't generate namespaces for the tests.

Ultimately it was decided to be non-blocking.